pkg/tfvars/powervs/powervs.go

```diff
@@ -44,6 +61,7 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 	//  --
 	cfg := &config{
 		IBMCloud:             sources.IBMCloud,
+		VPCRegion:            powervsRegionToIBMRegion[sources.IBMCloud.Region],
```

Comment:
technically, should this go into the IBMCloud struct? That struct is also a little funny to me now that we have this VPCRegion var. If it contains cloud-specific credential-type things, should we rename it from IBMCloud to something more generic, since, it holds the Power VS region? and maybe move the VPC (which is more of an IBM Cloud value) into it?

Reply:
I'll do a revision where we remove the IBMCloud struct and just pass each individually. The struct doesn't really get us much in the way of readability, so there's not much reason to keep it. Let me know what you think of that instead.

Reply:
yah i'm fine with removing that struct and just setting each value